docs/Developers/Writing_Documentation.rst

```diff
@@ -28,7 +28,7 @@ You should run this command before trying to build the documentation:
   pip3 install -r docs-requirements.txt
 
 You can test the documentation locally through simply running
-``make html`` in the root directory. This generates
+``python3 setup.py docs`` in the root directory. This generates
```

Comment:
Run the command and then check this paragraph..

Reply:
Yes, the command works

Reply:
You better write them in a code block.

Reply:
@ayan-b Is the change fine?

Reply:
Looks good. :+1:

Reply:
.. and then check this paragraph